### DISCUSSION TITLE
`Lint/EmptyBlock` can have some "false positives"

### DISCUSSION THREAD

**Discussion by stdedos** [rubocop/rubocop#13354] [Q&A] [answered]
e.g. using `File.open('a', 'w') { }` can be done to truncate a file in a safe mode.

Ofc, in this case, there is the `File.truncate('a', 0)`, but it can be considered a more "exotic" command.

**Comment by stdedos** [comment]
... And it won't work if the file does not exist currently.

But `File.open('a', 'w') { }` does work

**Comment by dvandersluis** [comment, marked as answer]
`File.open('a', 'w') { nil }` is accepted and more explicit IMO.
